src/traffic-collector/azext_traffic_collector/custom.py

```diff
@@ -9,6 +9,86 @@
+                })
+                # Throw error if bandwidth less than 1G
+                err_msg = """CollectorPolicy can not be created because circuit has bandwidth less than 1G.
+                Circuit size with a bandwidth of 1G or more is supported."""
+                # Provider Port Circuits
+                if (
+                    'serviceProviderProperties' in circuit
+                    and circuit['serviceProviderProperties']['bandwidthInMbps']
+                    and circuit['serviceProviderProperties']['bandwidthInMbps'] < 1000
+                ):
+                    raise InvalidArgumentValueError(err_msg)
+                # Direct Port Circuits
+                if 'bandwidthInGbps' in circuit and circuit['bandwidthInGbps'] < 1:
+                    raise InvalidArgumentValueError(err_msg)
+
+
+class CollectorPolicyUpdate(_CollectorPolicyUpdate):
+    def pre_operations(self):
+        args = self.ctx.args
+        if (
+            has_value(args.ingestion_policy)
+            and has_value(args.ingestion_policy.ingestion_sources)
+        ):
+            ingestionsources = args.ingestion_policy.ingestion_sources
+            for source in ingestionsources:
+                # Get ExpressRoute Circuit
+                subscription, resourceGroup, circuitName = parseResourceId(str(source.resource_id))
+                circuit = _ShowER(self)(command_args={
+                    "subscription": subscription,
+                    "resource_group": resourceGroup,
+                    "name": circuitName
+                })
+                # Throw error if bandwidth less than 1G
+                err_msg = """CollectorPolicy can not be created because circuit has bandwidth less than 1G.
+                Circuit size with a bandwidth of 1G or more is supported."""
+                # Provider Port Circuits
+                if (
+                    'serviceProviderProperties' in circuit
+                    and circuit['serviceProviderProperties']['bandwidthInMbps']
+                    and circuit['serviceProviderProperties']['bandwidthInMbps'] < 1000
+                ):
+                    raise InvalidArgumentValueError(err_msg)
+                # Direct Port Circuits
+                if 'bandwidthInGbps' in circuit and circuit['bandwidthInGbps'] < 1:
+                    raise InvalidArgumentValueError(err_msg)
+
+
+def parseResourceId(resourceId):
+    subscription = resourceId[resourceId.find('subscriptions/') +
+                              len('subscriptions/'):resourceId.rfind('/resourceGroups')]
+    resourceGroupName = resourceId[resourceId.find('resourceGroups/') +
+                                   len('resourceGroups/'):resourceId.rfind('/providers/Microsoft.Network')]
+    circuitName = resourceId[resourceId.find('Microsoft.Network/expressRouteCircuits/') +
+                             len('Microsoft.Network/expressRouteCircuits/'):]
+    return subscription, resourceGroupName, circuitName
```

Comment:
You can use `parse_resource_id` function to parse resource ID. For example:

https://github.com/Azure/azure-cli-extensions/blob/597bf58d54788f66bf96910daf4b46a838b9d80e/src/spring-cloud/azext_spring_cloud/_buildservices_factory.py#L107-L112

Reply:
Done